### PR TITLE
Added initial foundational services content

### PR DESCRIPTION
**Pull request by Agelb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+necessary skills and intuition for both cloud platforms in general and AWS specific services.
+
+## Learning AWS
+AWS has tons of services, and it can be overwhelming when starting. Knowing where to look and what to look for
+is one of the best things you can do to give yourself the best chance of success. AWS provides documentation about 
+each of their services, including user guides, whitepapers, and blogs. These three resources will provide almost 
+all the information that's available on a given service. If there are missing or poorly explained sections of AWS' 
+documentation, you can actually open a pull request on [their public docs repo](https://github.com/awsdocs), check the
+[AWS forums](https://forums.aws.amazon.com/index.jspa?categoryID=1), or check out stack overflow. However,
+note that AWS services are changing and evloving constantly, and info on stack overflow may be dated or incorrect.
```
Comment (ChallenHB) [inline README.md]: > evloving

Comment (Agelb) [reply README.md]: you've never heard of evloving before?

File: README.md [hunk 2]
```diff
@@ -1 +1,59 @@
...
+#### EC2
+EC2 is the service that supplies compute resources. AWS handles this with the concept of *instances*, which are 
+essentially VMs that are managed by AWS. A single EC2 instance provides a virtual machine with a set amount of resources    
+(CPU, RAM, Network bandwidth). We will skip over disk space for now, as disk and volumes are a large subject on their own 
+within AWS. The core idea here is that anything within AWS that uses compute, i.e. any software that runs, is running 
+on EC2.
+
+#### VPC
+Virtual Private Clouds are the networking services within AWS. A single VPC is roughly analagous to a data center.
+Compute resources must deployed _within_ a VPC - they can't just exist in a void. The VPC service is a collection of
```
Comment (ChallenHB) [inline README.md]: Should be "must be"

File: README.md [hunk 3]
```diff
@@ -1 +1,59 @@
...
+  
+#### S3
+The Simple Storage Service (S3) is an object store. This services provides data persistence that can be accessed 
+over the internet. On the surface it may look and feel like an SFTP server or filesystem, but it really is just a 
+general purpose storage system that stores objects and metadata about those objects. An object can be anything - 
+config files, images, jars, blobs, etc.
+
+#### EBS  
+Elastic Block Storage is block storage that can be attached to EC2 instances. This is different from S3, as EBS volumes
+can function as filesystems and boot volumes for compute resources. EBS volumes are their own entities, and can
```
Comment (ChallenHB) [inline README.md]: can be

File: README.md (hunk 1 above)
Comment (ChallenHB) [reply README.md]: Is that a new javascript framework?

**Pull request merged**